src/lib.sw

```diff
@@ -8,6 +9,7 @@ dep storage;
 dep b512;
 dep address;
 dep chain;
+dep option;
```

Comment:
Can you move this up near the top? Any stdlib modules that use it have to be added below this entry in lib.rs

Reply:
it could also get done in the feature to use option or result, so not a big deal.

Reply:
Fixed in [147de3d](https://github.com/FuelLabs/sway-lib-std/pull/68/commits/147de3de9a4e319ef7979af5cc993543ac00fb05)